lib/chainweb/pact/json_request_body.ex

```diff
@@ -1,5 +1,6 @@
 defprotocol Kadena.Chainweb.Pact.JSONRequestBody do
```

Comment:
@santiagoboterokommit as it is using the protocol outside of the request types, we probably have to rename and even relocate this module